packages/pn-personafisica-webapp/public/locales/it/recapiti.json

```
@@ -107,9 +107,8 @@
       },
       "step_3": {
         "title": "La tua email per ricevere avvisi sulle notifiche SEND",
-        "step-title": "Inserisci un recapito",
+        "step-title": "Inserisci la tua email",
         "content": "L’email dove ti avviseremo quando ricevi una comunicazione a valore legale per te su SEND.",
```

C'è un `per te` di troppo. 

```suggestion
        "content": "L’email dove ti avviseremo quando ricevi una comunicazione a valore legale su SEND.",
```